src/native/fs/fs_napi.cpp

```diff
@@ -34,6 +35,8 @@ namespace noobaa
 
 DBG_INIT(0);
 
+static int (*dlsym_gpfs_fcntl)(gpfs_file_t file, void* arg);
```

Comment:
Oh we forgot to initialize this pointer to null -
```suggestion
static int (*dlsym_gpfs_fcntl)(gpfs_file_t file, void* arg) = 0;
```